openlibrary/components/AuthorIdentifiers.vue

```diff
@@ -78,7 +78,7 @@ export default {
 
             // We use $set otherwise we wouldn't get the reactivity desired
             // See https://vuejs.org/v2/guide/reactivity.html#Change-Detection-Caveats
-            this.$set(this.assignedIdentifiers, this.selectedIdentifier, this.inputValue);
+            this.$set(this.assignedIdentifiers, this.selectedIdentifier, this.inputValue.trim());
```

Comment:
The more vue-way to do this would be to add it to the `v-model` in the html as `v-model.trim="blah"`. Then it auto-trims when it reads the value :) https://vuejs.org/v2/guide/forms.html#trim

Reply:
This is no longer needed now :)